### DISCUSSION TITLE
Route Groups do not work within [locale] folder

### DISCUSSION THREAD

**Discussion by BearCooder** [vercel/next.js#60939] [Help] [answered]
### Summary

Hello I have a strange issue. It seems that [Route Groups](https://nextjs.org/docs/app/building-your-application/routing/route-groups) do not work with [next-intl](https://next-intl-docs.vercel.app/)?

My following structure:
```
app/
 [locale]/
  (dashboard)/
    customers/
      page.tsx
    summary/
      page.tsx
   layout.tsx

```

Now when I want to access the customers page on `http://localhost:3000/customers` or `http://localhost:3000/summary` I get a 404 not found error.  I have to use `http://localhost:3000/dashboard/customers` to access the page.  But when I try it without the [locale] then it works as expected and I can access `http://localhost:3000/customers` without dashboard. When I put it all back into [locale] folder it does not work again?

I am not sure what causes this issue? I looked at the documentation in next intl  and next.js and couldnt find anything related to this.

**Comment by joulev** [comment, marked as answer]
`[locale]` is a dynamic route and you are required to put something there, else it will 404.

So you need something like `http://localhost:3000/en/customers`, then `params.locale` is `en`.

In your example, `http://localhost:3000/dashboard/customers` works because in that case, `params.locale` is `dashboard`. This part is not related to the route group – you can put it as `http://localhost:3000/anything-you-want/customers` and it still works.

When you go to `http://localhost:3000/customers`, Next.js will look for `/[locale]/page.tsx` or `/customers/page.tsx` (including route groups if any) – you don't have these files so it 404's.
